analytics/views/stats.py

```diff
@@ -124,7 +124,7 @@ def stats_for_remote_realm(
 @require_server_admin_api
 @has_request_variables
 def get_chart_data_for_realm(
-    request: HttpRequest, user_profile: UserProfile, realm_str: str, **kwargs: Any
+    request: HttpRequest, /, user_profile: UserProfile, realm_str: str, **kwargs: Any
```

Comment:
Although we need the `/` within the definition of the decorator, we don’t need it on the individual views, since `def f(request: HttpRequest, user_profile: UserProfile)` is a valid subtype of `def f(request: HttpRequest, /, user_profile: UserProfile)`.

Reply:
Oh I see, we do need it on views that take `**kwargs`.